resources/warnings.json

```diff
@@ -11276,8 +11276,8 @@
     "url": "https://www.jenkins.io/security/advisory/2022-03-15/#SECURITY-1886",
     "versions": [
       {
-        "lastVersion": "1.5",
-        "pattern": ".*"
+        "lastVersion": "282.v79ca_e079d1b_1",
+        "pattern": "1[.][0-5]|244[.]v27c8a_2e50a_34|269[.]v214f74360b_3a_|282[.]v79ca_e079d1b_1(|[.-].*)"
```

Comment:
```suggestion
        "pattern": "(1|244|269|282)(|[.-].+)"
```

Lack of grouping the version numbers negates the purpose of the `(|[.-].+)` suffix.

Reply:
Thanks very much!  I have much to learn about regex pattern matching.  Fixed in 3bf7b744dd82c35c4e3197c40a2eb7d46cffd2fb